samples/debugmalloc/host/Makefile

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+include ../../config.mk
+
+CFLAGS=$(shell pkg-config oehost-$(COMPILER) --cflags)
+LDFLAGS=$(shell pkg-config oehost-$(COMPILER) --libs)
+INCDIR=$(shell pkg-config oehost-$(COMPILER) --variable=includedir)
+
+build:
+	@ echo "Compilers used: $(CC), $(CXX)"
+	oeedger8r ../debugmalloc.edl --untrusted \
+		--search-path $(INCDIR) \
+		--search-path $(INCDIR)/openenclave/edl/sgx
```

Comment:
Why is this sample SGX only?   I didn't see anything in the sample that is specific to SGX.

Reply:
This is true for all samples, since they need to include platform.edl in the sgx directory. See helloworld sample below.
https://github.com/openenclave/openenclave/blob/10186a4db8e0ec187a404366c9af142c2e4a9d25/samples/helloworld/enclave/CMakeLists.txt#L9